### PR TITLE
Develop 

### PR DESCRIPTION
**Pull request by nicolasmarino99**
Awaiting for reply.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,419 @@
...
+        <img src="img/bob-ross-cover.png" alt="">
+        <a href="#">Where Are All the Bob Ross Paintings? We Found Them.</a>
+        <img src="img/merlin.jpg" alt="">
+        <a href="#">Need Etiquette Tips for Cannabis? For Starters, Don’t Call It ‘Marijuana’ or ‘Weed’</a>
+        <img src="img/snapshot.jpg" alt="">
+        <a href="#">When a Dating Dare Leads to Months of Soul Searching</a>
+      </div>
+    </aside>
+-->
+
```
Comment (samgaco) [inline index.html:290]: You have these two sections commented, which should be on the right side of the website.

File: styles/styles.css [hunk 2]
```diff
@@ -0,0 +1,458 @@
+/*
+  border: 2px dashed rgba(146, 0, 165, 0.5);
+  background-color: rgba(146, 0, 165, 0.15);
+  background-image: url("data:image/svg+xml;base64,PHN2ZyB4bWxucz0naHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmcnIHhtbG5zOnhsaW5rPSdodHRwOi8vd3d3LnczLm9yZy8xOTk5L3hsaW5rJyB3aWR0aD0nNicgaGVpZ2h0PSc2Jz4KICA8cmVjdCB3aWR0aD0nNicgaGVpZ2h0PSc2JyBmaWxsPScjZWVlZWVlJy8+CiAgPGcgaWQ9J2MnPgogICAgPHJlY3Qgd2lkdGg9JzMnIGhlaWdodD0nMycgZmlsbD0nI2U2ZTZlNicvPgogICAgPHJlY3QgeT0nMScgd2lkdGg9JzMnIGhlaWdodD0nMicgZmlsbD0nI2Q4ZDhkOCcvPgogIDwvZz4KICA8dXNlIHhsaW5rOmhyZWY9JyNjJyB4PSczJyB5PSczJy8+Cjwvc3ZnPg==");
+  background-repeat: repeat;
+*/
+
```
Comment (samgaco) [inline styles/styles.css:7]: It's better not to have commented code.

**Review by samgaco — CHANGES_REQUESTED** [review]
Hi dream team _(readme file)_ :laughing: 

The project looks great :muscle:  although there are some things to address before proceeding;

* First and foremost, the branch has a conflict in the README file, so you should resolve it before submitting for a code review again.
* The New York times logo in the center of the navigation bar should be a link.
* The New York times logo at the top left corner of the footer should be a link.
* There should be a _go to homepage_ link on the top right corner of the footer
* The sections **More in Space & Cosmos** and **Editors’ Picks** should have the titles as links.
* It lacks the two sections and the right side of the site;

> signup
> picks
> 


* All images and videos should be enclosed by a `<figure>` tag with a `<figcaption> `for the description. You can find more information about it here: https://www.w3schools.com/tags/tag_figcaption.asp

* The `<figcaption>` and `<figure>` tags should be also used for the left side of additional links. Following this structure:

```
<figure>
  <img src="image.jpg" alt="img">
  <figcaption>Title of the article.</figcaption>
</figure>
```

* Any section including paragraphs in your project should be enclosed within the` <section> `tag. In the case of the "Editor's picks" and the "Sign up" button they should also include the `<aside>` tag. Learn more about it here: https://www.w3schools.com/html/html5_semantic_elements.asp (you seem to have addressed this part already, but the code is commented in the pull request)

That would be it. Please :pray: commit these changes and submit again for a code review. :muscle:

**Review by bolah2009 — APPROVED** [review]
# Congratulations Team! :tada:

You've done an awesome Job :muscle:

#### Important: I strongly recommend updating your readme file with more details about the project, live version link, assignment link and screenshot of features implemented among other things you would like people visiting the Github link of this project to know about.

Good luck with your next project
Keep it up :star2:

Cheers
Bola

**Pull request merged**